hack/import-restrictions.yaml

```diff
@@ -82,6 +83,7 @@
   - "./vendor/k8s.io/client-go/tools/remotecommand"
   allowedImports:
   - k8s.io/apimachinery
+  - k8s.io/apiserver/pkg/util/trace
```

Comment:
this is forbidden. Please revert. We cannot depend on eachother.

Reply:
https://github.com/kubernetes/kubernetes/pull/73209 :)